pipeline/metadata/caida_ip_metadata.py

```diff
@@ -37,6 +35,24 @@
 AS_TO_ORG_HEADER = "# format:aut|changed|aut_name|org_id|opaque_id|source"
 
 
+class CaidaIpMetadataInterface:
+  """Interface for an CAIDA IP Metadata lookup database."""
+
+  def __init__(
+      self,
+      date: datetime.date,
+      cloud_data_location: str,
+      allow_previous_day: bool,
+  ) -> None:
+    pass
+
+  def lookup(
+      self, ip: str
+  ) -> Tuple[Optional[str], int, Optional[str], Optional[str], Optional[str],
```

Comment:
These tuples are hard to follow. Perhaps define a named tuple?
https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
added named tuples are return types